src/apps/backend/modules/comment/internal/comment-util.ts

```diff
@@ -0,0 +1,16 @@
+import { Comment } from '../types';
+import { CommentDB } from './store/comment-db';
+
+export const convertCommentDBToComment = (commentDb: CommentDB): Comment => ({
+  id: commentDb._id.toString(),
+  task: commentDb.task.toString(),
+  account: commentDb.account,
```

Comment:
This is not correct commentDb.account can be an object ID or it can be an object. You should handle both cases